tests/integration/migration/morse_claim_supplier_test.go

```diff
@@ -353,3 +354,7 @@ func (s *MigrationModuleTestSuite) TestClaimMorseSupplier_ErrorMinStake() {
 		).Error(),
 	).Error())
 }
+
+// TODO_MAINNET_MIGRATION(@bryanchriswhite): Add coverage for non-custodial cases:
```

Comment:
If we're not going to do it in this PR, I don' think it'll happen.

- If you really do plan to do it, can you create the named test w/ a t.skip
- If not, remove.